README.md

```diff
@@ -1,12 +1,64 @@
-# OCF Data Sampler
+# ocf-data-sampler
+ 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-5-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-[![ease of contribution: easy](https://img.shields.io/badge/ease%20of%20contribution:%20easy-32bd50)](https://github.com/openclimatefix/ocf-meta-repo?tab=readme-ov-file#overview-of-ocfs-nowcasting-repositories) 
 
-A repo for sampling from weather data for renewable energy prediction
+[![tags badge](https://img.shields.io/github/v/tag/openclimatefix/ocf-data-sampler?include_prereleases&sort=semver&color=FFAC5F)](https://github.com/openclimatefix/ocf-data-sampler/tags)
+[![ease of contribution: easy](https://img.shields.io/badge/ease%20of%20contribution:%20easy-32bd50)](https://github.com/openclimatefix#how-easy-is-it-to-get-involved) 
 
-## Contributors ✨
+**ocf-data-sampler** contains all the infrastructure needed to create batches and feed them to our models, such as [PVNet](https://github.com/openclimatefix/PVNet/). The data we work with is usually too heavy to do this on the fly, so that's where this repo comes in: handling steps like opening the data, selecting the right samples, normalising and reshaping, and saving to and reading from disk.
```

Comment:
I think this is a good short intro to the library, two comments:

1) The word infrastructure might make things a bit confusing since infrastructure usually relates to a specific meaning in software development e.g. the services defined in our ocf-infrastructure repo so perhaps worth rewording slightly 
2) Is it worth mentioning weather/energy data somewhere so it's clear that this is what is being sampled

Reply:
Good points, will do

Reply:
Made some changes, lmk what you think!